catalog/installWebSocket.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# use the command line interface to install websocket package.
+#
+: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=$WHISK_SYSTEM_AUTH
+
+SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
+CATALOG_HOME=$SCRIPTDIR
+source "$CATALOG_HOME/util.sh"
+
+echo Installing WebSocket package.
+
+createPackage websocket \
+    -a description "Utilities for communicating with WebSockets"
```

Comment:
Another option would be to make this "send" action part of some other package (which? I have no idea) and rename the action to something like "websocketSend". That being said, I believe the option of adding this to an existing package was already discussed in #393.